src/EnergyPlus/EarthTube.cc

```diff
@@ -566,44 +576,98 @@ namespace EarthTube {
 
 			}
 
-			InsideDewPointTemp = PsyTdpFnWPb( OutHumRat, OutBaroPress );
+			CalcEarthTubeHumRat( Loop, NZ );
+			
+		}
+
+	}
 
-			if ( EarthTubeSys( Loop ).InsideAirTemp >= InsideDewPointTemp ) {
-				InsideEnthalpy = PsyHFnTdbW( EarthTubeSys( Loop ).InsideAirTemp, OutHumRat );
-				// Intake fans will add some heat to the air, raising the temperature for an intake fan...
-				if ( EarthTubeSys( Loop ).FanType == IntakeEarthTube ) {
-					if ( EAMFL( NZ ) == 0.0 ) {
-						OutletAirEnthalpy = InsideEnthalpy;
-					} else {
-						OutletAirEnthalpy = InsideEnthalpy + EarthTubeSys( Loop ).FanPower / EAMFL( NZ );
-					}
-					EarthTubeSys( Loop ).AirTemp = PsyTdbFnHW( OutletAirEnthalpy, OutHumRat );
+	void
+	CalcEarthTubeHumRat(
+		int const Loop, // EarthTube number (index)
+		int const NZ // Zone number (index)
+	)
+	{
+		
+		// SUBROUTINE INFORMATION:
+		//       AUTHOR         Kwang Ho Lee
+		//       DATE WRITTEN   November 2005
+		//       MODIFIED       Rick Strand, June 2017 (made this a separate subroutine)
+		//       RE-ENGINEERED  na
+		
+		// PURPOSE OF THIS SUBROUTINE:
+		// This subroutine determines the leaving humidity ratio for the EarthTube
+		// and calculates parameters associated with humidity ratio.
+		
+		// METHODOLOGY EMPLOYED:
+		// na
+		
+		// REFERENCES:
+		// na
+		
+		// Using/Aliasing
+		// na
+		
+		// Locals
+		// SUBROUTINE ARGUMENT DEFINITIONS:
+		// na
+		
+		// SUBROUTINE PARAMETER DEFINITIONS:
+		// na
+		
+		// INTERFACE BLOCK SPECIFICATIONS
+		// na
+		
+		// DERIVED TYPE DEFINITIONS
+		// na
```

Comment:
@RKStrand we have been deleting unused comments in headers. I suggest removal. Also, please post a plot of results so reviewers have an easier time of understanding these changes.